From 49c3d0893091fb1bb6b92639e59a72203be0bc4a Mon Sep 17 00:00:00 2001
From: Damien Regad <dregad@mantisbt.org>
Date: Fri, 14 Nov 2014 22:55:47 +0100
Subject: [PATCH] Fix #17870: XSS in adm_config_report.php

This is the correct fix for this issue, using string_attribute() to
escape the variable. Thanks to Paul Richards for pointing this out.
---
 adm_config_report.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/adm_config_report.php b/adm_config_report.php
index 79ace81..d762491 100644
--- a/adm_config_report.php
+++ b/adm_config_report.php
@@ -480,7 +480,7 @@ function print_option_list_from_array( $p_array, $p_filter_value ) {
 	</td>
 	<td>
 		<input type="text" name="config_option"
-			value="<?php echo $t_edit_option; ?>"
+			value="<?php echo string_attribute( $t_edit_option ); ?>"
 			size="64" maxlength="64" />
 	</td>
 </tr>
